src/openassetio-python/package/openassetio/__init__.py

```diff
@@ -65,7 +65,7 @@
 -----------------
 
 The documentation for OpenAssetIO can be found here:
-   https://openassetio.github.io/OpenAssetIO.
+   https://docs.openassetio.org/OpenAssetIO.
```

Comment:
Nit: Do you want a full stop at the end of this, as the URL is on a line of it's own?

Reply:
Good spot, removed, took the liberty of force-pushing the update.